content/english/java-basics/activity-1.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Print Statements and Comments"
+description: "Introduce println() methods and comments in Java."
+date: 2020-07-08T00:00:00Z
+---
+
+### 1. Let the Program Speak! 
+
+Let’s try to print `Hello World` out by typing `System.out.print("Hello World");` 
+
+As you might have guessed, the program prints out whatever you typed in the parenthesis. And, each line of code ends with `;`.
+
+After, add more statements to print numbers by putting numbers in the parenthesis (i.e. `System.out.print(2020)`) or a symbol by putting one symbol between 2 single quotes `'` (i.e. `System.out.print('@')`).
+
+<iframe height="600px" width="100%" src="https://repl.it/@nuevofoundation/HelloWorld?lite=true#Main.java" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
+
+{{% notice tip %}}
+#### Do you notice that all the statements you typed end up printing everything in one line?
+Sometimes we want to enter a new line when typing. 
+And, to do that, we type a slightly different code `System.out.println();` 
+
+(Note: we type `println` instead of `print` to create a new line after)
+
+Give it a try and make all the statements in different lines above!
+{{% /notice %}}
+
+{{% notice tip %}}
+#### How to print multiple symbols in one line of code?
+We learned that to print symbols we put them around 2 single quotes `'`, but how do we print multiple symbols in one line?
+
+We can simple put them between 2 double quotes `"` (i.e. `System.out.print("@ , - h A #");`)
+
+{{% /notice %}}
+
+{{% notice tip %}}
+#### !! Important !! Sometimes you can't just put characters in between " "
+When putting characters in between `" "` in a print statement, sometimes it's easy to confuse the computer on what characters to print.
+
+For example, how do we tell computer to print `"`? 
+
+If you type `System.out.print(""");`, you will get an error! because the computer can't identify where the text ends!
+
+Instead, certain characters need to be <b>escaped</b> by adding a `\` in front of it.
+
+     System.out.print("\"");  // this prints out "
```

Comment:
If you type this for code blocks, (ideally) the website will color format the code block with the correct Java syntax:

```java
System.out.print("\"");
```